SourceX/DiabloUI/diabloui.cpp

```diff
@@ -124,7 +124,9 @@ BOOL EndDialog(HWND hDlg, INT_PTR nResult)
 
 BOOL SetWindowPos(HWND hWnd, HWND hWndInsertAfter, int X, int Y, int cx, int cy, UINT uFlags)
 {
+#ifndef USE_SDL1
 	SDL_SetWindowPosition(window, X, Y);

```

Comment:
Maybe implement this with DUMMY for now?

Reply:
Done